docs/write_tests.rst

```diff
@@ -0,0 +1,131 @@
+How to Write Tests for Parsons Connectors
```

Comment:
Is this doc linked from How to Build a Connector?  We should probably stick a quick link in the 'adding tests' section.